Polyhedron/demo/Polyhedron/Scene_c3t3_item.cpp

```diff
@@ -287,6 +287,35 @@ bool Scene_c3t3_item::load_binary(std::istream& is)
     return false;
 }
 
+void Scene_c3t3_item::compute_bbox() const
+{
+  if (isEmpty())
+    _bbox = Bbox();
+  else
+  {
+    bool bbox_init = false;
+    CGAL::Bbox_3 result;
+    for (Tr::Cell_handle c : c3t3().cells_in_complex())
+    {
+      if (bbox_init)
+      {
+        Tr::Vertex_handle v = (c->vertex(0) != c3t3().triangulation().infinite_vertex())
+                             ? c->vertex(0) : c->vertex(1);
+        result = result + v->point().bbox();
+      }
+      else
+      {
+        for(int i = 0; i < 4; ++i)
+          result = c->vertex(i)->point().bbox();
+        bbox_init = true;
+      }
```

Comment:
The for-loop seems strange. At the end of it, `result` will be equal to `c->vertex(3)->point().bbox()` no matter what are the other points of the cell.

Note also that, since CGAL-4.8, the default `Bbox_3` is a valid empty box (with intervals [+∞, -∞] on all three coordinates). So, as far as I understand, `bbox_init` is not useful.